helm-charts/agentqna/gaudi-values.yaml

```diff
@@ -20,6 +20,9 @@ vllm:
   VLLM_SKIP_WARMUP: true
   shmSize: 16Gi
   extraCmdArgs: ["--tensor-parallel-size", "4", "--max-seq_len-to-capture", "16384", "--enable-auto-tool-choice", "--tool-call-parser", "llama3_json"]
+  securityContext:
+    runAsUser: 0
```

Comment:
Is the `root` user used because deployment does not know what user can access /dev/accel/* device files?

In that case, please see: https://kubernetes.io/blog/2021/11/09/non-root-containers-and-devices/

Reply:
> Is the `root` user used because deployment does not know what user can access /dev/accel/* device files?
> 
> In that case, please see: https://kubernetes.io/blog/2021/11/09/non-root-containers-and-devices/

Nope. This is vllm-gaudi specific issue. tgi-gaudi runs well with non-root user on the same k8s node. I believe I've already  done the steps mentioned in the above blog

Reply:
> This is vllm-gaudi specific issue. tgi-gaudi runs well with non-root user on the same k8s node.

Could you file a bug against Habana vLLM fork, if there isn't one yet, and add link to it as comment e.g. to `vllm/gaudi-values.yaml`?

Reply:
Bug is filed in upstream, https://github.com/HabanaAI/vllm-fork/issues/929. Will add the comment in following PRs. I'd like to get this merged in first to unblock incoming dependent PRs